ImGuizmo.cpp

```diff
@@ -2096,10 +2096,12 @@ namespace IMGUIZMO_NAMESPACE
          // find new possible way to move
          vec_t gizmoHitProportion;
          type = GetMoveType(op, &gizmoHitProportion);
+#ifdef IMGUIZMO_MOUSE_CAPTURE_ON_HOVER
```

Comment:
If you want to add a #idef, you have to define it by default to not break backward compatibility.